tests/foreman/api/test_provisioningtemplate.py

```diff
@@ -244,16 +244,14 @@ def test_positive_build_pxe_default(self, tftpboot, module_target_sat):
                 r.raise_for_status()
                 rendered = r.text
             else:
-                rendered = module_target_sat.execute(f'cat {template["path"]}').stdout.splitlines()[
-                    0
-                ]
+                rendered = module_target_sat.execute(f'cat {template["path"]}').stdout
 
             if template['kind'] == 'iPXE':
                 assert f'{module_target_sat.hostname}/unattended/iPXE' in r.text
             else:
                 assert (
-                    rendered == f'{settings.server.scheme}://'
-                    f'{module_target_sat.hostname} {template["kind"]}'
+                    f'{settings.server.scheme}://{module_target_sat.hostname} {template["kind"]}'
+                    in rendered
```

Comment:
I'm not against this but how is it related the rest of this PR?

Reply:
yes, its unrelated, but in my last refactor PR I missed it, so updating it rn, looks non-blocking

Reply:
I'd prefer unrelated changes to be a separate PR **or** a separate commit, can you do that?